ops/charm.py

```diff
@@ -226,9 +207,9 @@ class StartEvent(HookEvent):
 
     This event is triggered immediately after the first
     :class:`ConfigChangedEvent`. Callback methods bound to the event should be
-    used to ensure that the charm’s software is in a running state. Note that
-    the charm’s software should be configured so as to persist in this state
-    through reboots without further intervention on Juju’s part.
+    used to ensure that the charm's software is in a running state. Note that
```

Comment:
These all get picked up - even though I do prefer the correct character we've discussed this previously, and Sphinx corrects them automatically for the docs, so it's only if you're reading this in the file itself that you see `'`. If I recall correctly we agreed that we should standardise on `'` so it seems ok to include that in this PR.